Library/Homebrew/test/test_apply.rb

```diff
@@ -0,0 +1,27 @@
+require "testing_env"
+require "patch"
+
+class ApplyTests < Homebrew::TestCase
```

Comment:
@mikemcquaid There's already a <code>test_patch.rb</code> and <code>test_patching.rb</code>, which were the basis for the apply tests:
https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/test/test_patch.rb
https://github.com/Homebrew/homebrew/blob/master/Library/Homebrew/test/test_patching.rb